js/start.js

```diff
@@ -263,15 +265,19 @@ let ownFollowingFailed;
 let exchangeRatesFetch;
 let walletBalanceFetch;
 let walletBalanceFetchFailed;
+let searchProvidersFetch;
+let searchProvidersFetchFailed;
 
 function fetchStartupData() {
   ownFollowingFetch = !ownFollowingFetch || ownFollowingFetch ?
     app.ownFollowing.fetch() : ownFollowingFetch;
   exchangeRatesFetch = exchangeRatesFetch || fetchExchangeRates();
   walletBalanceFetch = !walletBalanceFetch || walletBalanceFetch ?
     app.walletBalance.fetch() : walletBalanceFetch;
+  searchProvidersFetch = !searchProvidersFetch || searchProvidersFetch ?
```

Comment:
You're working off of an old master here. These lines have been updated. For example:

```
walletBalanceFetch = !walletBalanceFetch || walletBalanceFetch ?
```

has become:

```
walletBalanceFetch = !walletBalanceFetch || walletBalanceFetchFailed ?
```

and similarly:

```
searchProvidersFetch = !searchProvidersFetch || searchProvidersFetch ?
```

needs to be:

```
searchProvidersFetch = !searchProvidersFetch || searchProvidersFetchFailed ?
```